symengine/utilities/teuchos/Teuchos_RCP.hpp

```diff
@@ -733,6 +733,7 @@ Teuchos::rcp_static_cast(const RCP<T1>& p1)
 #endif
 }
 
+
```

Comment:
I noticed later that rest of the functions in this file have two empty line in between them. That's why I added the new line.